cli/src/main/scala/com/ibm/sparktc/sparkbench/workload/exercise/Sleep.scala

```diff
@@ -3,30 +3,143 @@ package com.ibm.sparktc.sparkbench.workload.exercise
 import com.ibm.sparktc.sparkbench.workload.{Workload, WorkloadDefaults}
 import com.ibm.sparktc.sparkbench.utils.GeneralFunctions._
 import org.apache.spark.sql.{DataFrame, SparkSession}
+import breeze.stats.distributions.{Poisson, Rand}
+import com.ibm.sparktc.sparkbench.utils.SparkBenchException
 
 case class SleepResult(
-                      name: String,
-                      timestamp: Long,
-                      total_runtime: Long
+                        name: String,
+                        timestamp: Long,
+                        total_runtime: Long
                       )
 
 object Sleep extends WorkloadDefaults {
+
+  def optionAnyToOptionLong(a: Option[Any]): Option[Long] = a match {
```

Comment:
map?

Reply:
Agreed. I took out this method and replaced it with just .map(any2long)